src/lib.rs

```diff
@@ -127,6 +210,8 @@ impl Default for PropertyType {
 #[derive(Encode, Decode, Clone, PartialEq, Eq)]
 pub enum PropertyValue {
     None,
```

Comment:
Why is `None` an option here? If its an artifact of the need for default constructability then remove that, we should not have default constructibility either.

Reply:
One can make a similar but weaker argument about `PropertyValue::None`. None is not an actual value for any of the types. None is only a possible value in the context of satisfying a schema or not. The direct way to model that would be in `ClassPropertyValue`, which models this directly, so it would be

 ```Rust
 struct ClassPropertyValue {
value: Option<PropertyValue>
}
```
but I don't know if it worth changing this now.

Reply:
Substrate requires a default value for enums that should be encoded/decoded (for example used on UI). That's why I need to add a default value for both enums: `PropertyType` and `PropertyValue`

Reply:
Ok, lets just stick with this for now.